File: AppMovilGestionCreditos1/lib/models/sesion_dto.dart
class SesionDTO {
  int id;
  String? nombreApellidos;
  String? correo;
  String? rolDescripcion;

  SesionDTO({
    this.id = 0,
    this.nombreApellidos,
    this.correo,
    this.rolDescripcion,
  });

  factory SesionDTO.fromJson(Map<String, dynamic> json) {
    return SesionDTO(
      id: json['id'] ?? 0,
      nombreApellidos: json['nombreApellidos'],
      correo: json['correo'],
      rolDescripcion: json['rolDescripcion'],
    );
  }

  Map<String, dynamic> toJson() => {
    'Id': id,
    'NombreApellidos': nombreApellidos,
    'Correo': correo,
    'RolDescripcion': rolDescripcion,
  };
}

File: ApiCredito2-main/GestionIntApi/DTO/SesionDTO.cs
using Microsoft.AspNetCore.Mvc;

namespace GestionIntApi.DTO
{
    public class SesionDTO 
    {
        public int Id { get; set; }
        public string? NombreApellidos { get; set; }
        public string? Correo { get; set; }
        public String? RolDescripcion { get; set; }

        public string Token { get; set; }
    }
}
